### ISSUE TITLE
Custom scrollbar on entire website

### ISSUE THREAD

**Issue by thomasbnt** [forem/forem#7414]
**Is your feature request related to a problem? Please describe.**

Adding a custom scrollbar for entire website. 
But having seen the scss files of the source code, I don't know where to put this little piece of code. Maybe in the `_colors.scss`?

**Describe the solution you'd like**
I'd like to make a small change on the scrollbar. Will follow the colors themes and blend in with the background.

| Minimal light theme | Dark theme |Pink theme | Hacker theme |
|-:|:-:|:-:|:-:|
|![dev to_ (1)](https://user-images.githubusercontent.com/14293805/79867426-9b515b00-83de-11ea-91e2-b9faad038ccd.png)|![dev to_settings_ux](https://user-images.githubusercontent.com/14293805/79867499-bae88380-83de-11ea-9c88-26bfd81f08e6.png)|![dev to_settings_ux (1)](https://user-images.githubusercontent.com/14293805/79867522-c176fb00-83de-11ea-9784-2bd51b0fb880.png)|![dev to_settings_ux (2)](https://user-images.githubusercontent.com/14293805/79867536-c9369f80-83de-11ea-8be9-d464339d7aa8.png)|

```javascript
// Scrollbar 
::-webkit-scrollbar {
  width: 9px;
}
::-webkit-scrollbar-track {
  background: transparent;
}
::-webkit-scrollbar-thumb {
  background: var(--base-40);
}
::-webkit-scrollbar-thumb:hover {
  background: var(--base-60); 
}
```

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by thomasbnt** [comment]
No news about that? :cake:

**Comment by cmgorton** [comment]
@ludwiczakpawel @nickytonline any thoughts on this?

**Comment by thomasbnt** [comment]
Thanks @cmgorton for relaunch this issue :octopus:

**Comment by ludwiczakpawel** [comment]
Hey,

There're some concerns I have in regards to custom scrollbars, accessibility related as well. You can read a bit more about this here: https://adrianroselli.com/2019/01/baseline-rules-for-scrollbar-usability.html

But overall, I'd love to know a bit of reasoning, why would we/site/community benefit from these custom scrollbars and not the browser's native ones.

**Comment by thomasbnt** [comment]
@ludwiczakpawel Helo ! 👋 
I thought that adding the customs scrollbar to the website would be cleaner. I have the impression that with the classic bars, it's from another era. The color, the little buttons and the gray background.

<details>
<summary>Image with the default scrollbar</summary>

![image](https://user-images.githubusercontent.com/14293805/101359609-cf5d8b80-389c-11eb-9644-6998c60fb423.png)

</details>

**Comment by ludwiczakpawel** [comment]
Hey, thanks for this. I tried to evaluate whether it's needed from user's perspective right now. Because of the issues with accessibility I linked to earlier I think we will put it on hold for now and maybe circle back later in the future when we have better idea and concept on how to handle scrollbars more holistically.

I'm closing this issue but I leave the conversation open if there's something more that needs to be discussed here. Once again thank you very much for this idea!
